docs/README.md

```diff
@@ -46,5 +46,6 @@ Please refer to: https://github.com/nasa/cumulus/blob/master/CONTRIBUTING.md for
 ### Data Cookbook
 
 * [About Cookbooks](data-cookbooks/about-cookbooks.md)
+  * [Collections, Providers, Schemas, and Rules](data-cookbooks/setup.md)
```

Comment:
Moved the link under the "About Cookbooks" tag. This structure felt a bit more intuitive to me.